packages/openapi-v3/src/controller-spec.ts

```diff
@@ -363,6 +363,7 @@ function processSchemaExtensions(
    *  You cannot provide BOTH a defnintion AND one of these keywords.
    */
   /* istanbul ignore else */
+  // eslint-disable-next-line no-prototype-builtins
   const hasOwn = (prop: string) => schema?.hasOwnProperty(prop);
```

Comment:
Have you considered to fix the code per https://eslint.org/docs/rules/no-prototype-builtins instead of silencing the linter?

We have a check on L347 to handle the case when `schema` is not set, I think the following should work well:

```suggestion
  const hasOwn = (prop: string) => Object.prototype.hasOwnProperty.call(schema, prop);
```